src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -33,8 +34,8 @@ public void testCallCheck() {
 	//Test fails, because of implementation based on IntLinkedList
 	@Test
 	public void testIsFull() {
-		DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
-		for (int i = 0; i < STACK_CAPACITY; i++) {
+		DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer(new IntArrayStack());
+		for (int i = 0; i < STACK_CAPACITY - 1; i++) {

```

Comment:
Is it a correct approach to fixing errors showed by tests? 
If STACK_CAPACITY is 12 then 12 elements should be added to the stack.
Errors are still within the project.